async_upnp_client/ssdp.py

```diff
@@ -66,9 +121,15 @@ def decode_ssdp_packet(data: bytes, addr: str) -> Tuple[str, CaseInsensitiveDict
     email_headers = email.message_from_bytes(header_lines)
     headers = CaseInsensitiveDict(**dict(email_headers.items()))
 
+    # adjust some headers
+    if "location" in headers:
+        headers["location"] = get_adjusted_url(headers["location"], addr)
```

Comment:
This modifies the header in place to append the zone/scope needed for link local access. Don't think it breaks anything. But seems you previously added extras with an "_XXX". Do we want to do that here too?

Reply:
Perhaps the original location, e.g., `_location_original`.

Though I do not know if this is really needed. I'm going by your judgement/experience.

Reply:
For link local it is. You must have scope on host, otherwise the local system don't know which interface to use for outgoing. Same address can exist on multiple physical links.

Sadly seem even chrome is broken with this. New versions of curl work good. I did not test your http access code. Probably should :)